src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/OpenDocumentGenerator.cs

```diff
@@ -106,25 +112,11 @@ private void ProjectManager_Changed(object? sender, ProjectChangeEventArgs args)
                 }
 
             case ProjectChangeKind.DocumentAdded:
-                {
-                    var newProject = args.Newer.AssumeNotNull();
-                    var documentFilePath = args.DocumentFilePath.AssumeNotNull();
-
-                    if (newProject.TryGetDocument(documentFilePath, out var document))
-                    {
-                        // We don't enqueue the current document because added documents are initially closed.
```

Comment:
The change is because we have too many moving pieces at the moment. When I changed things so that newly added documents stopped guessing which was the right project to be added to, it meant we didn't start generating when a document moved from the misc files project to the right one. Since the `EnqueueIfNecessary` already had a check for only generating open documents, this explicit lack of work made no sense to me, and seems to me like it was only valid because the "guess which project to go in" was guessing correctly most of the time.